pkg/cli/server/server.go

```diff
@@ -187,6 +187,8 @@ func run(app *cli.Context, cfg *cmds.Server) error {
 	agentConfig.ServerURL = url
 	agentConfig.Token = token
 	agentConfig.DisableLoadBalancer = true
+	agentConfig.Rootless = true // let agent specify Rootless kubelet flags
```

Comment:
```suggestion
	agentConfig.Rootless = cfg.Rootless
```
I think this should be passing in the value from the command line, otherwise the rootless logic in `pkg/daemons/agent/agent.go` is always triggered when running the server (master nodes).

Reply:
:+1: fixed, thanks